peps/pep-0693.rst

```diff
@@ -60,6 +60,13 @@ Actual:
 - 3.12.0 candidate 3: Tuesday, 2023-09-19
 - 3.12.0 final:  Monday, 2023-10-02
 
+Bugfix releases
+---------------
+
+Actual:
+
+- 3.12.1: Thursday, 2023-12-07
+
 Subsequent bugfix releases every two months.
```

Comment:
Hello @jwodder,
I suppose this might be removed already.
```suggestion
```
Also, is it possible to add expected release dates, just like it was in #2915?

Reply:
There's another eight or so bugfix releases still to come, so we shouldn't remove this line yet unless expected release dates are added. 

On that topic, see https://discuss.python.org/t/pep-693-missing-future-python-3-12-release-dates/40372?u=hugovk.